src/ensembl/package.json

```diff
@@ -103,6 +103,7 @@
     "dotenv-webpack": "7.0.2",
     "eslint": "7.27.0",
     "eslint-config-prettier": "8.3.0",
+    "eslint-plugin-jest": "^24.3.6",
```

Comment:
Let's do the exact version; no need for the `^`.

Reply:
✅